mne/io/tests/test_raw.py

```diff
@@ -319,8 +319,8 @@ def _test_raw_reader(
     assert raw.__class__.__name__ in repr(raw)  # to test repr
     assert raw.info.__class__.__name__ in repr(raw.info)
     assert isinstance(raw.info["dig"], (type(None), list))
-    data_max = full_data.max()
-    data_min = full_data.min()
```

Comment:
@larsoner I did make this change in `test_raw` that I just want to make sure you are aware of.

Reply:
Yes I saw this one and I think it's fine :)